### PR TITLE
feat: add links ui

### PR DESCRIPTION
**Pull request by JoeTechx**

### CODE REVIEW

File: app/globals.css
Comment (evergreenx) [inline app/globals.css:1]: You should not define reusable styles in the global.css, tailwind handles that in the config 
[(https://tailwindcss.com/docs/configuration)](url)

**Review by evergreenx — CHANGES_REQUESTED** [review]
Check my comment

File: app/globals.css
Comment (JoeTechx) [reply app/globals.css:1]: ok boss

Comment (evergreenx) [reply app/globals.css:1]: no worries i will handle it

Comment (JoeTechx) [reply app/globals.css:1]: I have done it already

**Pull request merged**